test-framework/test-suites/integration/tests/report/test_report_host.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+
+@pytest.mark.usefixtures("revert_database")
+class TestReportHost:
+    def test_report_host_with_default_network_behaviour():
+        result = host.run(f'stack add host backend-0-0')
+        assert result.rc == 0
+        result = host.run(f'stack add host interface backend-0-0 interface=eth0 ip=10.10.11.11 network=private')
+        assert result.rc == 0
+        result = host.run(f'stack add host backend-0-1')
+        assert result.rc == 0
+        result = host.run(f'stack add host interface backend-0-1 interface=eth1 ip=10.10.11.15 network=private default=True')
+        assert result.rc == 0
+        result = host.run(f'stack report host')
```

Comment:
It's not enough to see the command didn't fail.  `stack report host` was already succeeding with bad output.  You should actually validate that the output of stack report host is actually what you expect (either a line is present or is not present, depending on what was desired).